website/content/api-docs/agent.mdx

```diff
@@ -725,3 +725,204 @@ $ curl -O -J \
 
 go tool trace trace
 ```
+
+
+## Fetch all scheduler worker's status
+
+The `/agent/schedulers` endpoint allow Nomad operators to inspect the state of
+a Nomad server agent's scheduler workers.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/agent/schedulers` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required |
+| ---------------- | ------------ |
+| `NO`             | `agent:read` |
+
+### Parameters
+
+This endpoint accepts no additional parameters.
+
+### Sample Request
+
+```shell-session
+$ curl \
+    https://localhost:4646/v1/agent/schedulers
+```
+
+### Sample Response
+
+```json
+{
```

Comment:
Incredibly nitpicky-nitpick: I think all our JSON response bodies in these docs have 2-space indentation.